website/src/_template/contributing.md

```diff
@@ -125,10 +122,8 @@ Even though bundled in this repo, the website is regarded as a separate project.
 ### Local previews
 
 1. `npm install`
-2. `npm run bootstrap`
-3. `cd website && npm install && cd ..`
-4. `npm start`
-5. Go to http://localhost:4000. Your changes in `/website` and `/packages/@uppy` will be watched, your browser will refresh as files change.
+2. `npm start`
+3. Go to http://localhost:4000. Your changes in `/website` and `/packages/@uppy` will be watched, your browser will refresh as files change.
```

Comment:
This got automatically overwritten for me, I think, we have to change the `.github/CONTRIBUTING.md` instead, as it says on the top of this file.